src/acvp_capabilities.c

```diff
@@ -993,6 +993,11 @@ static ACVP_RESULT acvp_validate_sym_cipher_parm_value(ACVP_CIPHER cipher, ACVP_
         case ACVP_KAS_FFC_NOCOMP:
         case ACVP_KAS_KDF_ONESTEP:
         case ACVP_KAS_HKDF:
+        case ACVP_KAS_FFC_SSC:
```

Comment:
Do we still use the fulll ACVP_CIPHER enum in these places because TDES and AES are different sub-enums? Should we make them the same sym_ciph sub enum, or move towards making tdes and aes separate structures?

Reply:
Exactly, we could dig into that later and it may save a bunch of space on the sym APIs however it would also add more entries to AES and TDES. I didn't dig into much but it may end up being about equal.